src/components/bookmarkV2/index.js

```diff
@@ -86,6 +104,8 @@ class Bookmark extends React.Component {
       if (this.getFilterList().length === 0) this.setState({ isLoading: false });
       this.props.validateBookmark();
     }, 300);
+
+    if (e && e.target.value === '') this.resetListIndex();
```

Comment:
maybe validate that the `e.target` also exists
```suggestion
    if (e && e.target && e.target.value === '') this.resetListIndex();
```

Reply:
done